jaq-core/src/lib.rs

```diff
@@ -402,3 +404,93 @@ const LOG: &[(&str, usize, RunPtr, UpdatePtr)] = &[(
     |_, cv| box_once(Ok(debug(cv.1))),
     |_, cv, f| f(debug(cv.1)),
 )];
+
+#[cfg(feature = "format")]
+fn to_csv(v: Val, delimiter: u8) -> ValR {
+    use csv::{StringRecord, WriterBuilder};
+    match v.clone() {
+        Val::Arr(vs) => {
+            let mut writer = WriterBuilder::new()
+                .delimiter(delimiter)
+                .from_writer(vec![]);
+            let mut record = StringRecord::new();
+            for cell in &*vs {
+                match cell {
+                    Val::Str(s) => {
+                        record.push_field(s);
+                        Ok(())
+                    }
+                    _ => Err(Error::Str(cell.clone())),
+                }?;
```

Comment:
You could shorten this a lot by using `cell.as_str()`.

Reply:
This remark of mine is probably obsolete if you consider [my comment below](https://github.com/01mf02/jaq/pull/105#discussion_r1289774934).